src/host/tests/snapshots/plan-from.js.md

```diff
@@ -4840,18 +4840,7 @@ Generated by [AVA](https://avajs.dev).
         },
       },
       source: 'fork',
-      tests: [
-        {
-          filepath: 'tests/1.test.json',
-          log: [],
-          results: [],
-        },
-        {
-          filepath: 'tests/2.test.json',
-          log: [],
-          results: [],
-        },
-      ],
+      tests: [],
```

Comment:
This is correct, the default test match pattern is `{,**/}test*` so `tests/1.test.json` and `tests/2.test.json` should never have been matched here (all further instances of this are this change)

Reply:
It's great to learn that this change is correcting previously-unreported errors! And while I fully support keeping these tests by correcting their expectations, *only* doing that feels like a missed opportunity. What do you think about adding new tests which correctly reflect the original intent (e.g. with a pattern like `{,**/*}test*`)?

Reply:
This test already exists.  Specifically `for (const testPattern of [undefined, '*.json', 'tests/*'])` - these corrected results are only in the `undefined` testPattern - the `*.json` and `tests/*` both still match this

Reply:
Correction: only the `tests/*` pattern finds the file (which is correct) - I incorrectly stated `*.json` also was (which would be wrong)